client/web/src/hooks/node-data.ts

```diff
@@ -46,16 +46,26 @@ export type UserProfile = {
   ProfilePicURL: string
 }
 
-export type NodeUpdate = {
-  AdvertiseRoutes?: string
-  AdvertiseExitNode?: boolean
+export type SubnetRoute = {
+  Route: string
+  Approved: boolean
+}
+
+export type NodeUpdaters = {
```

Comment:
I kinda like this set of distinct methods for updating aspects of the current node, but maybe add some basic docs here?